Backend/src/main/resources/db/changelog/changeset/backend-changeset-0.0.2.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+<changeSet id="test-change-2" author="Rohit Saini">
+    <createTable tableName="USER_DETAILS">
+        <column name="user_id" type="int"></column>
```

Comment:
explore more. add constraints. create sequences which you can refer it in the code